Tests/CarthageKitTests/AlgorithmsSpec.swift

```diff
@@ -2,8 +2,10 @@ import CarthageKit
 import Nimble
 import Quick
 
-class AlgorithmsSpec: QuickSpec {
-	override func spec() {
+// swiftlint:disable:this file_length
```

Comment:
I've introduced a new `.swiftlint.yml` configuration just for the tests. I updated both yaml files to accommodate this change.